src/ColorSchemes.jl

```diff
@@ -21,29 +21,26 @@ export ColorScheme,
        loadcolorscheme,
        findcolorscheme
 
-"""
-    ColorScheme struct with members `colors`, `category`, and `name`.
-"""
-struct ColorScheme
-    colors::Vector{C} where {C <: Colorant}
-    category::AbstractString
-    notes::AbstractString
-end
 
 """
     ColorScheme(colors, category, notes)
 
 Create a ColorScheme from an array of colors. You can also name it, assign a
 category to it, and add notes.
 
-```
+```julia
 myscheme = ColorScheme([Colors.RGB(0.0, 0.0, 0.0), Colors.RGB(1.0, 1.0, 1.0)],
     "custom", "twotone, black and white")
 ```
 
 """
-ColorScheme(colors, category::AbstractString) = ColorScheme(colors, category, "")
-ColorScheme(colors) = ColorScheme(colors, "general", "")
+struct ColorScheme
+    colors::Vector{C} where {C <: Colorant}
+    category::AbstractString
+    notes::AbstractString
+    
+    ColorScheme(colors::Vector{<: Colorant}, category::AbstractString = "", notes::AbstractString = "") = new(colors, category, notes)
+end
 
```

Comment:
```suggestion

Base.getindex(cs::ColorScheme, i::AbstractFloat) = get(cs, i)
Base.getindex(cs::ColorScheme, i::AbstractVector{<: AbstractFloat}) = get(cs, i)

Reply:
@cormullion would you want this sort of a feature?  You could then do `cs[0.5]` in a colorscheme, which is a small convenience - but comes in handy sometimes.

Reply:
Looks cool! Anything is good as long as it's not slower or more complicated! :)